x/concentrated-liquidity/lp.go

```diff
@@ -43,6 +43,8 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 	if err := validateTickRangeIsValid(pool.GetTickSpacing(), lowerTick, upperTick); err != nil {
 		return 0, sdk.Int{}, sdk.Int{}, sdk.Dec{}, time.Time{}, err
 	}
+	amount0Desired := tokensProvided.AmountOf(pool.GetToken0())
+	amount1Desired := tokensProvided.AmountOf(pool.GetToken1())
```

Comment:
No, since if both are zero then the liquidityDelta that is calculated is zero. That being said, I think you are right if your point was we should catch this earlier, will add this now

Reply:
https://github.com/osmosis-labs/osmosis/pull/5119/commits/cbdc89814db9c94515e42522554b43346cdec3aa